package.json

```diff
@@ -58,7 +58,6 @@
     "lint": "eslint --cache --cache-location ./node_modules/.cache/eslint .",
     "prebuild": "rimraf dist",
     "predocs": "rimraf docs/pages/docs",
-    "prepare": "pnpm run build",
```

Comment:
it's weird to have a `pnpm add` fail because the build isn't working. I don't think `prepare` scripts are a good fit for this